kview/src/main/java/dev/ikm/komet/kview/fxutils/MenuHelper.java

```diff
@@ -126,7 +126,7 @@ public static void fireContextMenuEvent(ActionEvent actionEvent, Side side, doub
      * This method takes list of menuItems in 2D array form.
      * The 2nd array can have the following items in sequence: public NAME = 0; ENABLED = 1; STYLES = 2; ACTION = 3; GRAPHIC = 4;
      * @param menuItems
-     * @return
+     * @return contextMenu
```

Comment:
Can you elaborate with a sentence?

Reply:
added more details on the returned value.